WordPress/src/main/res/values/strings.xml

```diff
@@ -754,6 +761,7 @@
     <string name="add_account_blog_url">Blog address</string>
     <string name="wordpress_blog">WordPress blog</string>
     <string name="blogusername">blogusername</string>
+    <string name="wordpress_com_url" translatable="false">.wordpress.com</string>
```

Comment:
This is a minor point, but I think the string name should denote that the URL is prefixed with "." (down the road I can see us wanting to show the wordpress.com URL elsewhere).